src/GrantType/JwtBearer.php

```diff
@@ -3,6 +3,7 @@
 namespace CommerceGuys\Guzzle\Oauth2\GrantType;
 
 use GuzzleHttp\ClientInterface;
+use CommerceGuys\Guzzle\Oauth2\GrantType\GrantTypeBase;
```

Comment:
Looks like you didn't base this on `master`? This is restoring an incorrect line from an older version